src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -558,6 +563,10 @@ internal HttpRequestMessage BuildSetTriggersRequest()
 
             var hostname = _hostNameProvider.Value;
             var url = $"{protocol}://{hostname}/operations/settriggers";
+            if (_environment.IsKubernetesManagedHosting())
+            {
+                url = $"http://{ManagedKubernetesBuildServiceName}.{ManagedKubernetesBuildServiceNamespace}.svc.cluster.local:{ManagedKubernetesBuildServicePort}/operations/settriggers";
```

Comment:
What is the auth model for this request? In SetTriggers below, we're sending a token that the FE uses for authentication. Is this request also handled by the FE?

Reply:
Hi @mathewc - in this case request goes to the build service (kudulite) running in the same Kubernetes cluster as the function host, and the encryption key for the function app will also be accessible to build service, which is used to decrypt the token and pass the authentication successfully. The function app access token is created/updated through the geo-master control plane code as a Kubernetes secret and build service will have access to read the secret. So, this flow in Kubernetes doesn't hit the Antares front end. Hope this helps.

Reply:
Ok, so the token is actually required (not ignored) in your scenario - that's all I was trying to establish.

Reply:
Got it, yes that's right @mathewc.